client/webserver/site/src/js/markets.ts

```diff
@@ -955,9 +955,16 @@ export default class MarketsPage extends BasePage {
 
     this.updateRegistrationStatusView()
 
+    const showSection = (section: PageElement | undefined) => {
+      const elements = [page.registrationStatus, page.bondRequired, page.bondCreationPending, page.notRegistered]
+      for (const el of elements) {
+        Doc.setVis(el === section, el)
+      }
+    }
+
     if (market.dex.auth.effectiveTier >= 1) {
       const toggle = async () => {
-        Doc.hide(page.registrationStatus, page.bondRequired, page.bondCreationPending)
+        showSection(undefined)
```

Comment:
I finished bond posting and the order form has been displayed but "Fidelity bond accepted!" remained on the order form until I refreshed,

Reply:
That's what this line is supposed to do.. and the "Fidelity bond accepted" disappears for me. Maybe you didn't build the latest UI?